src/main/scala/castalia/Protocol.scala

```diff
@@ -1,11 +1,17 @@
 package castalia
 
+import castalia.model.{ResponseConfig, StubConfig}
 import spray.json.DefaultJsonProtocol
```

Comment:
Is this from `akka-http-spray-json`? Of just `spray-json`, then we can upgrade to the akka-http version